pages/en/learn/modules/publishing-node-api-modules.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: How to publish N-API package
-layout: docs.hbs
+title: How to publish Node-API package
+layout: learn.hbs
 ---
 
-# To publish N-API version of a package alongside a non-N-API version
+# How to publish Node-API version of a package alongside a non-Node-API version
 
 The following steps are illustrated using the package `iotivity-node`:
```

Comment:
I wonder if this package should be replaced - it was [archived in 2019](https://github.com/intel/iotivity-node)

Reply:
the archieved repository references: https://github.com/iotivity/iotivity-lite though i could not find a nodejs binding package for the newer repository

Reply:
@legendecas do you know if this is a problem? or is there a better example?

Reply:
there could be some better examples. I'm not really familiar with iotivitity or other node.js bindings available that could be used as a replacement in the documentation. I'm not really sure if it could be a problem, but since it serves as an example, and that the binding is still available on npmjs i believe it shouldn't be a problem when serving as an example to a reader.